src/algorithm/SORTPAIRS.cpp

```diff
@@ -40,6 +40,11 @@ SORTPAIRS::~SORTPAIRS()
 {
 }
 
+Index_type SORTPAIRS::getProblemSize() const
```

Comment:
Is there a reason to not do the implementation of these methods in the header file? Less code to maintain.

Reply:
I see what you mean. I put them in the cpp files because the rest of the logic for problem sizes is in the source files.

Reply:
I can see that perspective, but I don't think the extra code is worth it.

Reply:
Ok, Then I'll move this and iterations per rep into the headers.